src/exampleTypes/Number.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { observer } from 'mobx-react'
+import { Flex } from '../layout/Flex'
+import { exampleTypes } from 'contexture-client'
+import injectTreeNode from '../utils/injectTreeNode'
+
+export default injectTreeNode(
+  observer(({ tree, node, ...props }) => (
+    <Flex {...props}>
+      <input
+        className="contexture-search-box"
+        type="number"
+        value={node.min || ''}
```

Comment:
That was added because if it's null, react thinks it's uncontrolled: https://github.com/smartprocure/contexture-react/pull/28/commits/d818577621cd9733aac6b5742c085325f2fd1b82

`null` and `undefiined` didn't work, and 0 is not correct.